test/fuzz/lua_fuzzer/lua_fuzzer.cc

```diff
@@ -0,0 +1,38 @@
+extern "C"
+{
+#include <lua.h>
+#include <lualib.h>
+#include <lauxlib.h>
+}
+
+#include "lua_grammar.pb.h"
+#include "serializer.h"
+
+#include <libprotobuf-mutator/port/protobuf.h>
+#include <libprotobuf-mutator/src/libfuzzer/libfuzzer_macro.h>
+
+DEFINE_PROTO_FUZZER(const lua_grammar::LuaBlock &message)
+{
+	lua_State *L = luaL_newstate();
+	if (!L)
+		return;
+
+	std::string code = LuaBlockToString(message);
+
+	/* See https://luajit.org/running.html
+	 * https://github.com/tarantool/tarantool/issues/6782
+	 */
+	luaL_dostring(L, "jit.opt.start('hotloop=1')");
+	luaL_dostring(L, "jit.opt.start('hotexit=1')");
+	luaL_dostring(L, "jit.opt.start('recunroll=1')");
+	luaL_dostring(L, "jit.opt.start('callunroll=1')");
+
+	luaL_openlibs(L);
+	int status = luaL_loadbuffer(L, code.c_str(), code.size(), "fuzz_test");
+	if (status == 0)
+		lua_pcall(L, 0, 0, LUA_MULTRET);
+
+	std::cerr << "Code: " << code << std::endl;
```

Comment:
I used it because I though fuzzer could suppress outputs on std::cout. Added opportunity to display generated code on std::cout when environment variable 'LPM_DUMP_NATIVE_INPUT' is set to 'ON'.